README.md

```diff
@@ -1,11 +1,96 @@
-# Eppo JS SDK for Browser Applications
+# Eppo Javascript SDK
 
 [![](https://img.shields.io/npm/v/@eppo/js-client-sdk)](https://www.npmjs.com/package/@eppo/js-client-sdk)
+[![Test and lint SDK](https://github.com/Eppo-exp/js-client-sdk/actions/workflows/lint-test-sdk.yml/badge.svg)](https://github.com/Eppo-exp/js-client-sdk/actions/workflows/lint-test-sdk.yml)
 [![](https://img.shields.io/static/v1?label=GitHub+Pages&message=API+reference&color=00add8)](https://eppo-exp.github.io/js-client-sdk/js-client-sdk.html)
 [![](https://data.jsdelivr.com/v1/package/npm/@eppo/js-client-sdk/badge)](https://www.jsdelivr.com/package/npm/@eppo/js-client-sdk)
 
-This SDK is for client-side JS applications that run in a web browser. For server-side JS applications, use Eppo's [server-side Node JS SDK](https://github.com/Eppo-exp/node-server-sdk).
+[Eppo](https://www.geteppo.com/) is a modular flagging and experimentation analysis tool that supports a wide range of flagging use cases, from simple on/off toggles to highly personalized contextual bandits. Eppo's Javascript SDK is built to make assignments for single user client applications that run in a web browser. For the server side equivalent, use Eppo's [Node SDK](https://github.com/Eppo-exp/node-server-sdk). Before proceeding you'll need an Eppo account.
+
+## Installation
+
+```
+npm install @eppo/js-client-sdk
+```
+
+## Usage
+
+Initialization creates a singleton instance of Eppo's client and fetches the account's flag configuration, which is cached locally. Once initialized, assignments can be made anywhere in your app.
```

Comment:
Suggested rephrase -

"Once initialized, the SDK can assign variations from anywhere in your app."